.gitignore

```diff
@@ -132,6 +132,9 @@ client/galaxy/scripts/libs/*
 # Documentation build files.
 doc/build
 doc/schema.md
+doc/source/admin/config_logging_default_yaml.rst
+doc/source/api/api.rst
+doc/source/api/ts_api.rst
```

Comment:
If we now want to ignore these 2 files, then we should also remove them from the git repo, I think.